keylime/tenant.py

```diff
@@ -180,11 +180,21 @@ def init_add(self, args):
             vtpm_policy = args["vtpm_policy"]
         self.vtpm_policy = TPM_Utilities.readPolicy(vtpm_policy)
         logger.info(f"TPM PCR Mask from policy is {self.vtpm_policy['mask']}")
+        al_data = None
 
         # Read command-line path string allowlist
-        al_data = None
         if "allowlist" in args and args["allowlist"] is not None:
 
+            _policy_pcrs = list(self.tpm_policy.keys())
+            _policy_pcrs.remove('mask')
+
+            _protected_pcrs = [ config.IMA_PCR ]
+
+            for _pcr in _policy_pcrs :
+                if int(_pcr) in _protected_pcrs :
```

Comment:
You are looking though all the _policy_pcrs to compare against the single IMA_PCR. Is this for future purposes? Because it would seem easier to look the other way around, trying `if is config.IMA_PCR in _protected_pcrs`...

Reply:
Or maybe you have to do 
```
_policy_pcrs = [int(x) for x in _policy_pcrs]
if config.IMA_PCR in _policy_pcrs`:
```
or
```
_policy_pcrs = set([int(x) for x in _policy_pcrs])
is = set.intersection(set(_protected_pcrs), _policy_pcrs)
if len(is) > 0: ...
```